EIPS/eip-3403.md

```diff
@@ -59,7 +60,53 @@ TBD
 
 ## Test Cases
 
-TBD
+### Berlin costs
+
+With Berlin, and [EIP-2929](https://eips.ethereum.org/EIPS/eip-2929), the gas costs changes. Note, there is a difference between 'hot' and 'cold' slots.
+This table show the values assuming that all touched storage slots were already 'hot' (the difference being a one-time cost of 2000 gas).
```

Comment:
```suggestion
### 2929 Gas Costs

Note, there is a difference between 'hot' and 'cold' slots.
This table shows the values as of [EIP-2929](./eip-2929.md) assuming that all touched storage slots were already 'hot' (the difference being a one-time cost of 2000 gas).
```
In an effort to keep a strong schelling fence up to keep out hard fork coordination stuff from this repository, lets just refer to it as 2929 gas costs.